packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -275,6 +275,7 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
       title: 'Update profile',
       imageFormTitle: 'Profile image',
       imageFormSubtitle: 'Upload image',
+      destructiveActionSubtitle: 'Delete image',
```

Comment:
:question: Should this say "Delete" or "Remove"?

Reply:
Remove sounds better

Reply:
@SokratisVidros , @nikosdouvlis what do you think?

Reply:
Since the image can't be retrieved, I'd say delete is better here.

Reply:
Let's do Remove